db/migrate/20250123091551_add_unique_constraint_to_national_id.rb

```diff
@@ -0,0 +1,6 @@
+class AddUniqueConstraintToNationalId < ActiveRecord::Migration[7.0]
+  def change
+    add_index :person_details, :national_id, unique: true
+    add_index :person_details_audits, :national_id, unique: true
```

Comment:
Being this an audit table I am thinking it will not be neccessary to have it Unique in case of tracking cross matches
```suggestion

```